config/rbac/role.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   creationTimestamp: null
-  name: manager-role
```

Comment:
manager-role is generated via controller-gen, you can run cmd `make manifests`

Reply:
Fixed. Changed before so my new role will not interfere with the existing manager-role in the test cluster.